projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/facet-list/facet-list.component.ts

```diff
@@ -49,20 +59,35 @@ export class FacetListComponent {
   @Output() closeList = new EventEmitter();
 
   /** The list of all facet and values related to the products in the list */
-  facetList$: Observable<FacetList> = this.facetService.facetList$;
+  facetList$: Observable<FacetList> = this.facetService.facetList$.pipe(
+    tap((facetList) => {
+      // TODO: (CXSPA-7321) - Remove feature flag next major release
+      const isFeatureEnabled = this.featureConfigService?.isEnabled(
+        'a11yFacetsDialogFocusHandling'
+      );
+      if (!isFeatureEnabled) {
+        return;
+      }
+      this.handleDialogFocus(facetList.facets);
+    })
+  );
 
   iconTypes = ICON_TYPE;
 
   dialogFocusConfig: FocusConfig = {
     trap: true,
     block: true,
     focusOnEscape: true,
-    autofocus: 'cx-facet',
+    autofocus: 'cx-facet > button',
```

Comment:
More just a question, cx-facet is not focusable as-is so autofocus usually has no effect here?

Reply:
Yes, if you will open a demo page and open the filters dialog then nothing will be autofocused, I figured it shouldn't work that way so now the first facets will be focused automatically

Reply:
That's what I thought. Thanks for clarifying :)